src/Task_21/Person.java

```diff
@@ -0,0 +1,26 @@
+package Task_21;
+
+public class Person {
+
+    public String name;
+    public String surname;
+    public String sex;
+    public String age;
+
+    public Person(String name, String surname, String sex, String age){
```

Comment:
Try to avoid constructors with paramters if you have no real need for this.
It's better to use getters to initialize object after creation